package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "nest build",
+    "generate-swagger": "node ./dist/swagger/generate-swagger-file.js",
```

Comment:
This requires to build project into js first, may be able to add the build process into the command or use `ts-node` to run it directly

Reply:
I don't think it's necessary, it solves our needs, but can consider in the future.